packages/daemon/src/daemon.js

```diff
@@ -131,26 +131,25 @@ const makeEndoBootstrap = async (
 
   /**
    * @param {string} workerId512
-   * @param {import('./types.js').Terminator} terminator
+   * @param {import('./types.js').Context} context
    */
-  const makeIdentifiedWorkerController = async (workerId512, terminator) => {
+  const makeIdentifiedWorkerController = async (workerId512, context) => {
     // TODO validate workerId512
     const daemonWorkerFacet = makeWorkerBootstrap(workerId512);
 
-    const { reject: cancelWorker, promise: workerCancelled } =
+    const { promise: hardCancelled, reject: hardCancel } =
       /** @type {import('@endo/promise-kit').PromiseKit<never>} */ (
         makePromiseKit()
       );
-    cancelled.catch(async error => cancelWorker(error));
 
     const { workerTerminated, workerDaemonFacet } =
       await controlPowers.makeWorker(
         workerId512,
         daemonWorkerFacet,
-        Promise.race([workerCancelled, gracePeriodElapsed]),
+        Promise.race([hardCancelled, gracePeriodElapsed]),
       );
 
-    const terminate = async () => {
+    const softCancel = async () => {
```

Comment:
what's softCancel

Reply:
I renamed the couplet `gracefulCancel`/`forceCancel`. I think the former just means "respect the worker's termination grace period", but I won't add more documentation for right now.